src/app/wallet-ui-get/wallet-ui-get.component.html

```diff
@@ -0,0 +1,114 @@
+<div class="container text-center pb-0 mt-3 wallet">
+  <app-global-header
+    [showBackground]="true"
+    [showTitle]="true"
+    [showUsername]="true"
+  ></app-global-header>
+</div>
+<div class="pb-200" *ngIf="!showCredentialList">
+  <div class="container text-center pb-0 mt-20">
```

Comment:
@ak-ironix I think we dont need code for showing credentialList, Please remove it from the template and typescirpt code as well
cc: @snehal0904